homeassistant/helpers/aiohttp_client.py

```diff
@@ -279,10 +284,12 @@ def _async_close_websession(event: Event) -> None:
 
 @callback
 def _make_key(
-    verify_ssl: bool = True, family: socket.AddressFamily = socket.AF_UNSPEC
-) -> tuple[bool, socket.AddressFamily]:
+    verify_ssl: bool = True,
+    ssl_cipher: ssl_util.SSLCipherList = ssl_util.SSLCipherList.PYTHON_DEFAULT,
```

Comment:
Please make the new key the last item in the tuple

Reply:
Done